manim/mobject/number_line.py

```diff
@@ -68,6 +68,29 @@ class NumberLine(Line):
     kwargs : Any
         Additional arguments to be passed to :class:`~.Line`.
 
+    Examples
+    --------
+    .. manim:: NumLineExample
+        :save_last_frame:
+        class NumLineExample(Scene):
```

Comment:
The docs currently aren't rendering the example, I think it's because of this missing space here.
```suggestion
        :save_last_frame:
        
        class NumLineExample(Scene):
```